### DISCUSSION TITLE
TBH: Beginner Problems & Beginner Questions :)

### DISCUSSION THREAD

**Discussion by TylerHarrison** [stlink-org/stlink#1236] [Q&A (resolved)] [answered]
Hello all!

I'm an aspiring electronic engineering student, with a growing interest in embedded systems; I love the work everyone has done here; supper fun! I have a question.

1) Where is the best place to ask questions related to the use of this tool, and not to its development?

Again, thank you for your time! :) 

Cheers,
Tyler

**Comment by Nightwalker-87** [comment, marked as answer]
Hi. I recommend you to have a look at our documentation and the related tutorial which can be found in the subfolder `/doc`.
Further specific questions can be addressed in this discussion category.

**Reply by TylerHarrison** [reply]
Thank you kindly!
